jenkins/automation/alex.groovy

```diff
@@ -143,7 +153,7 @@ pipeline {
               script {
                   def useEmailList = ''
                   if ( params.environment == 'prod') {
-                    useEmailList = 'shailesh.vaidya@seagate.com, priyank.p.dalal@seagate.com, john.bent@seagate.com, venkatesh.k@seagate.com'
+                    useEmailList = 'nilesh.govande@seagate.com, basavaraj.kirunge@seagate.com, ivan.tishchenko@seagate.com, john.bent@seagate.com, venkatesh.k@seagate.com'
```

Comment:
You can remove nilesh govande as he is not in cortx

Reply:
Thanks Gaurav. I have removed him from the list